### DISCUSSION TITLE
certificate is not yet valid

### DISCUSSION THREAD

**Discussion by cancan101** [percy/cli#554] [Support] [answered]
Getting this error:
```
[percy:client] Creating a new build... (28ms)
[percy:core:install] Downloading Chromium 885264...
[percy:cli:snapshot] Error: certificate is not yet valid
    at TLSSocket.onConnectSecure (node:_tls_wrap:1530:34)
    at TLSSocket.emit (node:events:394:28)
    at TLSSocket._finishInit (node:_tls_wrap:944:8)
    at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:725:12) (1526ms)
Error: Command failed with exit code 1: npx percy snapshot -v snapshots.yml
    at makeError (/home/runner/work/app/app/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/home/runner/work/app/app/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
```

**Comment by Robdel12** [comment, marked as answer]
Hey @cancan101! That likely means the certificate isn't valid _or_ the systems time is incorrect? Googling that error suggests the certificates time is wrong or the systems time is wrong

**Reply by cancan101** [reply]
Ok I think the clock was the issue as I was tinkering with the clock in order so solve the dynamic date issue (https://github.com/percy/cli/discussions/504#discussioncomment-1343795).

Is there any way to tell tell Percy-cli to _just_ download chromium (e.g. something like `npx percy snapshot --chromium-only`), as the download should be run w/o mucking with time? Right now I am using: `npx percy snapshot --debug snapshots.yml || echo "downloaded"` and then I run the real test with `faketime 2021-08-17 20:30:00 npx percy snapshot snapshots.yml`.

Alternatively some way to specify the prefix to the chromium command that is used so that just chromium itself it run with the modified time. I see [here](https://docs.percy.io/docs/cli-configuration#a-complete-config), `executable` however I don't want to specify the entire path.

~This page: https://docs.percy.io/docs/skipping-chromium-download also seems broken though it is linked to from: https://docs.percy.io/docs/cli-overview#advanced~

**Reply by Robdel12** [reply]
Ah that makes sense. Sorry about the docs, those were linking to the wrong doc in the first place -- they're fixed now 🙌🏼

You can set `PERCY_POSTINSTALL_BROWSER=true` and then run an install. That will install the browser in a postinstall step and I think that avoids this whole issue for you: https://docs.percy.io/docs/caching-asset-discovery-browser-in-ci
